File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/UserSetting.java
package org.fossasia.susi.ai.rest.responses.susi;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by meeera on 30/6/17.
 */

public class UserSetting {
    @SerializedName("session")
    @Expose
    private Session session;
    @SerializedName("settings")
    @Expose
    private Settings settings;

    /**
     * Gets session
     *
     * @return the session
     */
    public Session getSession() {
        return session;
    }

    /**
     * Gets settings
     *
     * @return the settings
     */
    public Settings getSettings() {
        return settings;
    }
}

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/UserSetting.kt
package org.fossasia.susi.ai.rest.responses.susi

import com.google.gson.annotations.SerializedName

/**
 * Created by meeera on 30/6/17.
 */

data class UserSetting (

    /**
     * Gets session
     *
     * @return the session
     */
    @SerializedName("session")
    val session: Session? = null,

    /**
     * Gets settings
     *
     * @return the settings
     */
    @SerializedName("settings")
    val settings: Settings? = null
)
